### ISSUE TITLE
Disable auto creation of dashboards Zabbix/Linux Server

### ISSUE THREAD

**Issue by daianmartinho** [alexanderzobnin/grafana-zabbix#422]
### I'm submitting a ...
- [ ] Bug report
- [ ] Feature requestdd
- [X] Question / Support request

Can you disable the automatic creation of Zabbix / Linux server dashboards? I have to delete them from all organizations every time i make update

**Comment by alexanderzobnin** [comment]
@daianmartinho can you describe a reason for this?

**Comment by daianmartinho** [comment]
I use the plugin as datasource. I have many organizations, and they dont have to see zabbix servers informations. More then that, they are not alowed to see. So those dashboards became trash, and i have to delete them always. If someone need them, they could add by zabbix plugin menu.

Em 10 de jul de 2017 7:57 AM, Alexander Zobnin <notifications@github.com> escreveu:

@daianmartinho<https://github.com/daianmartinho> can you describe a reason for this?

**Comment by FireHelmet** [comment]
@alexanderzobnin I'm in the same case.

Thanks for your work and support.

**Comment by ptempier** [comment]
Yup, same request, having the graphs installed by defautl is nice for the first time.
But after it get annoying, specially with a lot of organisation.

**Comment by julianoromani** [comment]
@alexanderzobnin same request. 
Thanks.

**Comment by alexanderzobnin** [comment]
Zabbix Server dashboard was only dashboard that enables plugin menu in Grafana side bar. So now it's not showing there anymore. So this closes #203 and #647 as well.

![screenshot from 2018-11-15 11-30-21](https://user-images.githubusercontent.com/4932851/48540020-f0c03900-e8c9-11e8-82a1-1588219f0819.png)
